service/chartconfig/v1/resource/chart/create.go

```diff
@@ -29,6 +30,12 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		if err != nil {
 			return microerror.Mask(err)
 		}
+		defer func() {
+			err := os.Remove(tarballPath)
+			if err != nil {
+				r.logger.LogCtx(ctx, "level", "error", "message", fmt.Sprintf("deletion of %q failed: %v", tarballPath, err))
```

Comment:
Lets please separate the message from the stack like in other places. See https://github.com/giantswarm/operatorkit/blob/1428f611909b304d221954fc09ddfbbe68beb3eb/framework/framework.go#L167.

Reply:
Done.